lightstep/resource_dashboard.go

```diff
@@ -48,6 +48,31 @@ func getUnifiedQuerySchemaMap() map[string]*schema.Schema {
 						Type:     schema.TypeString,
 						Optional: true,
 					},
+					"y_axis_scale": {
+						Type:     schema.TypeString,
+						Optional: true,
+						ValidateFunc: validation.StringInSlice([]string{
+							"linear",
+							"log",
+							"symlog",
+						}, false),
+					},
+					"y_axis_log_base": {
+						Type:     schema.TypeInt,
+						Optional: true,
+						ValidateFunc: validation.IntInSlice([]int{
+							2,
+							10,
+						}),
+					},
+					"y_axis_min": {
+						Type:     schema.TypeFloat,
```

Comment:
float matches what we do for the [deprecated version](https://github.com/lightstep/terraform-provider-lightstep/blob/c6e0c48de3cfb56ae66d8658cbb6a779ae9581e5/lightstep/resource_metric_dashboard.go#L257) of these options, i assume we do this so these values can get very large